src/components/Dashboard/Empty/index.test.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+import { render, screen } from '@testing-library/react';
+
+import DashboardEmpty from '.';
+
+describe('DashboardEmpty', () => {
+  const dataTestId = 'dashboard-empty';
+  it('renders DashboardEmpty and its components', () => {
+    render(<DashboardEmpty data-test-id={dataTestId} />);
```

Comment:
As you just use this const once, could you move it into the `it()`?
```suggestion
  it('renders DashboardEmpty and its components', () => {
  	const dataTestId = 'dashboard-empty';
    render(<DashboardEmpty data-test-id={dataTestId} />);
```

Reply:
@hoangmirs updated, thanks 🙆‍♂️